sources/ecs-gpu-init/cmd/ecs-gpu-init/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/NVIDIA/go-nvml/pkg/nvml"
+)
```

Comment:
How large is this 100 LOC Go binary on disk?

Reply:
Around 2M:

```bash
bash-5.1# ls -lh ecs-gpu-init
-rwxr-xr-x. 1 root root 2.1M Apr 27 18:07 ecs-gpu-init
```